Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
Not trying to be patronizing but, is it too much trouble to restart on Windows and test-build? (A simple msbuild.exe Project64.sln should do. No need to even open Visual Studio).

I would gladly settle for a warning about incoming build breaks beforehand.